src/Features/Core/Portable/Completion/CompletionItem.cs

```diff
@@ -109,6 +119,20 @@ public sealed class CompletionItem : IComparable<CompletionItem>
             return Create(displayText, filterText, sortText, properties, tags, rules, displayTextPrefix: null, displayTextSuffix: null);
```

Comment:
nit: either change this to pass all default arguments explicitly, or not passing them at all, the inconsistency here bothers me a little bit :)

Reply:
1. "or not passing them at all".  I cannot do that, otherwise this method would just be calling into itself.
2. "either change this to pass all default arguments explicitly".  I'm not sure what this means.